spec/oncocore_genetics.txt

```diff
@@ -13,16 +13,16 @@ Concept:		LNC#81247-9 "Master HL7 genetic variant reporting panel"
 Description:	"Genetic analysis summary report"
 				SubjectOfRecord is type PatientSubjectOfRecord
 1..1			Category is OBSCAT#GE
-				Observation is type GeneticRegionAnalysis
+				Observation is type GeneticRegionAnalysisResult
```

Comment:
Discussed already but just for tracking - We might want to reconsider `Result` in `GeneticRegionAnalysisResult` name. The CGWG might not have intended for GeneticRegion to have a result, but rather characterize the region that was analyzed.  Even if GeneticVariantFound is specified as a member, the actual result for the analysis of what was tested is not in the CG-Genomics-Report FHIR R4 IG obs-region-studied profile.